float32 r_pX
float32 r_pY
float32 r_pZ
float32 l_pX
float32 l_pY
float32 l_pZ
float32 r_elbow_dir
float32 l_elbow_dir
float32 max_step
float32 motion_time

